dockstore-webservice/pom.xml

```diff
@@ -87,6 +87,11 @@
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
         -->
+        <dependency>
```

Comment:
I think you can just use Java time library now. Joda website itself says only use Joda pre-Java 8: https://www.joda.org/joda-time/ -- the Joda developer apparently worked on the JSR to update Java date/time.